fuzzing/private/oss_fuzz/package.bzl

```diff
@@ -35,6 +57,8 @@ def _oss_fuzz_package_impl(ctx):
             }}
             trap cleanup EXIT
             ln -s "$(pwd)/{binary_path}" "$STAGING_DIR/{base_name}"
+            mkdir "$STAGING_DIR/{binary_deps_dir}"
+            {binary_deps_script}
```

Comment:
Would it be possible to implement the logic of computing these files as part of this script?

Right now the logic of this action is split between shell script fragments and Starlark code, making the code hard to validate upon visual inspection.

It would be great to avoid forming a script out of Starlark-assembled snippets.

Reply:
I'm not good with shell scripts, which is probably why I try to do as much in Starlark as possible. Could you give me some hints how I could do more with the shell here? I guess I could use a bash loop instead of a Starlark one, but how would I go iterate over pairs `(runfile_path, runfile_name)` in Bash?

Reply:
I think you can use the `find` command to iterate over the entire directory tree. You can loop over the command output and run the `ln -s` command to create the symbolic links (you might need to explicitly create the intermediate directories, too, but I'm not 100% sure).

Let me know if you run into any issues - I'm happy to dig further into this.